pkg/server/commands/write.go

```diff
@@ -91,7 +99,7 @@ func (c *WriteCommand) validateWriteRequest(ctx context.Context, req *openfgav1.
 			if errors.Is(err, storage.ErrNotFound) {
 				return serverErrors.AuthorizationModelNotFound(modelID)
 			}
-			return err
+			return serverErrors.HandleError("", err)
```

Comment:
I greatly dislike having `command` files return grpc errors but that's a separate PR

Reply:
Yeah this will have to change everywhere if we go that route of having commands return sentinel errors.